docs/dc-apps-performance-toolkit-user-guide-jira.md

```diff
@@ -309,14 +309,15 @@ After [Importing the main dataset](#importingdataset), you'll now have to pre-lo
 Do not close or interrupt the session. It will take about two hours to upload attachments to Elastic File Storage (EFS).
 {{% /note %}}
 
-### <a id="reindexing"></a> Re-indexing Jira Data Center (~1 hour)
+### <a id="reindexing"></a> Re-indexing Jira Data Center (~30 minutes)
 
 For more information, go to [Re-indexing Jira](https://confluence.atlassian.com/adminjiraserver/search-indexing-938847710.html).
 
 1. Log in as a user with the **Jira System Administrators** [global permission](https://confluence.atlassian.com/adminjiraserver/managing-global-permissions-938847142.html).
 1. Go to **![cog icon](/platform/marketplace/images/cog.png) &gt; System &gt; Indexing**.
-1. Select the **Lock one Jira node and rebuild index** option.
+1. Select the **Full re-index** option.
 1. Click **Re-Index** and wait until re-indexing is completed.
+1. Attach the screenshot to your DC HELP ticket.
```

Comment:
1. **Take a screenshot of the acknowledgment screen** displaying the re-index time and Lucene index timing.
1. Attach the screenshot to your DCHELP ticket.